### DISCUSSION TITLE
Salam salam salam

### DISCUSSION THREAD

**Discussion by sharabash** [quran/discussions#1] [Announcements]
السلام عليكم

Opening discussions

**Comment by roadiedev** [comment]
Waalaikum as-salaam wa rahmatullohi wa barakatuh!

**Reply by moabdulhakim** [reply]
وعليكم السلام

**Comment by JackHonak** [comment]
Assalam alikum, I am trying to add a reciter to the application. Is there anyone who can advise on the steps?
